.env.template

```diff
@@ -23,24 +26,35 @@ CSO_FILINGFACADE_USERNAME=
 CSO_SERVICEFACADE_URI=
 CSO_SERVICEFACADE_USERNAME=
 CSO_SERVICEFACADE_PASSWORD=
-KEYCLOAK_AUTH_SERVER_URL=
+CSO_FILE_SERVER_HOST=
+CSO_DEBUG_ENABLED=true
+
+CEIS_BASE_PATH=
+CEIS_USERNAME=BLANK?
+CEIS_PASSWORD=BLANK?
+
+KEYCLOAK_SSL_REQUIRED=none
+KEYCLOAK_RESOURCE=efiling-api
+KEYCLOAK_URL=
 KEYCLOAK_REALM=
-KEYCLOAK_RESOURCE=
 KEYCLOAK_CREDENTIALS_SECRET=
-KEYCLOAK_SSL_REQUIRED=
+KEYCLOAK_AUTH_SERVER_URL=
+EFILING_DEMO_KEYCLOAK_URL=
+EFILING_DEMO_KEYCLOAK_REALM=
+EFILING_DEMO_KEYCLOAK_CREDENTIALS_SECRET=
+
 BAMBORA_APIPASSCODE=
 BAMBORA_MERCHANTID=
+BAMBORA_PROFILE_URL=
+BAMBORA_HASHKEY=
+BAMBORA_PROFILE_SERVICE_VERSION=
+BAMBORA_URL_EXPIRY=
+
 SFTP_KNOWNHOSTS=
 SFTP_REMOTELOCATION=
 SFTP_PRIVATE_KEY=
+
 BCEID_SERVICE_URI=
 BCEID_SERVICE_USERNAME=
 BCEID_SERVICE_PASSWORD=
 BCEID_SERVICE_ONLINE_SERVICE_ID=
-CSO_FILE_SERVER_HOST=
-CSO_DEBUG_ENABLED=
-BAMBORA_PROFILE_URL=
-BAMBORA_HASHKEY=
-BAMBORA_PROFILE_SERVICE_VERSION=
-BAMBORA_URL_EXPIRY=
-CEIS_BASE_PATH=
```

Comment:
Where is the best place to document explanations, and in some cases examples, for the .env.template variables?

Reply:
I think our main readme should have it

Reply:
I'll start adding it to the readme